src/RadixDlt.NetworkGateway.PostgresIntegration/Models/PendingTransaction.cs

```diff
@@ -216,6 +216,8 @@ public void MarkAsCommitted(bool succeeded, DateTime timestamp)
     {
         Status = succeeded ? PendingTransactionStatus.CommittedSuccess : PendingTransactionStatus.CommittedFailure;
         CommitTimestamp = timestamp;
+        FailureReason = null;
```

Comment:
Hmm if it's a committed failure, we should ideally include the failure reason here?

Reply:
Precisely. This should be renamed to "most recent transient error/failure reason"